addons/escoria-core/game/core-scripts/esc_room.gd

```diff
@@ -110,13 +110,6 @@ func _ready():
 				player.update_idle()
 		escoria.object_manager.get_object("_camera").node.set_target(player)
 	
-	for n in get_children():
```

Comment:
Why did you remove this? How are the room object's registered then?

Reply:
This block only registers children node that are `ESCLocation`s and `is_start_location`.
All `ESCItem`s register by themselves in their own `_ready()` function.

Reply:
Ah right.